### ISSUE TITLE
Failure to add none spatial layer from DB Manager to map canvas

### ISSUE THREAD

**Issue by NyakudyaA** [qgis/QGIS#30532]
**Describe the bug**
DB Manager fails to load none spatial layer to the canvas

**How to Reproduce**

1. Create a PostgreSQL connection
2. Navigate to DB Manager and run a SQL query to create a none spatial layer
3. Refresh your PostgreSQL connection and try to load the layer into QGIS

![shot](https://user-images.githubusercontent.com/2510900/60582065-66f5ab00-9d88-11e9-9616-f2f7c2615170.png)

**QGIS and OS versions**
```
An error has occurred while executing Python code: 

AttributeError: 'PGTable' object has no attribute 'srid' 
Traceback (most recent call last):
  File "/usr/share/qgis/python/plugins/db_manager/db_tree.py", line 163, in addLayer
    layer = table.toMapLayer()
  File "/usr/share/qgis/python/plugins/db_manager/db_plugins/plugin.py", line 733, in toMapLayer
    uri = self.uri().uri(False)
  File "/usr/share/qgis/python/plugins/db_manager/db_plugins/plugin.py", line 716, in uri
    uri.setSrid(str(self.srid))
AttributeError: 'PGTable' object has no attribute 'srid'


Python version: 3.7.3 (default, Apr  3 2019, 05:39:12) [GCC 8.3.0] 
QGIS version: 3.8.0-Zanzibar Zanzibar, 11aff65 

Python Path:
/usr/share/qgis/python
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins
/usr/share/qgis/python/plugins
/usr/lib/python37.zip
/usr/lib/python3.7
/usr/lib/python3.7/lib-dynload
/home/admire/.local/lib/python3.7/site-packages
/usr/local/lib/python3.7/dist-packages
/usr/lib/python3/dist-packages
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins/inasafe
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins/inasafe/safe_extras/parameters
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins/inasafe/safe_extras
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins/mmqgis/forms
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins/joinmultiplelines
/home/admire/.local/share/QGIS/QGIS3/profiles/default/python/plugins
```

**Additional context**

OS: Ubuntu 18.04

**Comment by Gustry** [comment]
Duplicate of https://github.com/qgis/QGIS/issues/30260 and already fixed in 3.8 and master

**Comment by NyakudyaA** [comment]
Thanks @Gustry  will close this
